.ci/scripts/test.sh

```diff
@@ -210,6 +213,8 @@ else
   DOCKER_COMPOSE_FILE=docker-compose-all.yml
 fi
 
+ELASTIC_APM_ASYNC_HOOKS=${ELASTIC_APM_ASYNC_HOOKS:-true}
+
```

Comment:
REVIEW NOTE: This ensures that `ELASTIC_APM_ASYNC_HOOKS` is either "true" or "false", and not "" or "false". When it is the empty string, things are *fine*, but the agent logs:

```
{"log.level":"warn","@timestamp":"2021-06-22T23:22:55.871Z","log":{"logger":"elastic-apm-node"},"ecs":{"version":"1.6.0"},"message":"unrecognized boolean value \"\" for \"asyncHooks\""}
{"log.level":"warn","@timestamp":"2021-06-22T23:22:55.872Z","log":{"logger":"elastic-apm-node"},"ecs":{"version":"1.6.0"},"message":"unrecognized boolean value \"\" for \"asyncHooks\""}
```

which is just noise in test output.